content/blog/2025-02-03-submit_posit_conf/index.en.md

```diff
@@ -0,0 +1,77 @@
+
+## **Tips from Past Speakers**
+
+We know that recording a video can feel like a challenge, so we collated some advice from past speakers:
+
+📽  **Rachael Dempsey**: Go ahead and set a date and time on your calendar and block it off to record your video. 🗓 I had blocked off a Friday afternoon at 2pm. Give yourself a bit more time than you think you need. You might record yourself about 4+ times. [[Read more excellent advise from Rachael in this detailed LinkedIn post](https://www.linkedin.com/posts/rachaeldempsey_a-little-birdie-told-me-that-the-call-for-activity-7282863651554754563-Zy_I/).]
+
+🎤 **Shannon Pileggi**: Ask more than one person for feedback on your abstract and video content - everyone has a different and valuable point of view.Think about the way the principles of your content apply broadly to a general audience. [[Read Shannon's blog post talking about her experience of getting accepted to speak at Posit Conf here](https://www.pipinghotdata.com/posts/2024-10-24-iterating-to-achieve-my-first-accepted-positconf-talk/)]
+
+💡 **Nic Crane**: Check out the [Posit blog post which tells you what they're looking for](https://posit.co/blog/how-we-build-the-positconf-program/) and has examples of past submissions.  Before recording, I try to get myself pumped up with energy by doing a dummy run pretending my idea is the most interesting thing I've heard in my entire life, with exaggerated enthusiasm, so then the normal version I do immediately afterwards has a bit of bounce to it.
+
+## **What Tech Should You Use?**
+
+#### Simplest Options (Free & Easy):
+ - `Phone/Laptop Camera`: This is perfectly acceptable! Just ensure good lighting (face the light source) and a stable surface for your device. Most phones and laptops have built-in video recording capabilities.
+ - `QuickTime (Mac)`: If you have a Mac, QuickTime Player has a built-in screen recording feature. You can record your screen and your webcam simultaneously.
+ - `Zoom (App/Web)`: Zoom's built-in recording feature makes it easy to capture your screen and webcam simultaneously. Share your slides, practice your one-minute pitch, and download the MP4 file for submission
+
+#### Slightly Advanced (Free/Open Source):
+ - `OBS Studio (Windows, Mac, Linux)`: OBS Studio is free and open-source, offering more control over your recording, including screen capture, webcam input, and basic audio mixing. While powerful, it might have a small learning curve.
+ - `VLC Media Player (Windows, Mac, Linux)`: VLC can also be used for basic recording. It's a versatile player that can capture video from your webcam. 
+
+#### User Friendly (Paid):
+ - `Canva`: offers a user-friendly platform with recording capabilities. You can easily create slides with your key points and record yourself presenting them. Canva also provides editing features to polish your video.
+
+## **How to Structure Your Video**
+
+Here's a simple way to approach your **one-minute** video:
+
+### **1. Start with the Basics**
```

Comment:
When you look at the preview, the use of bolding and header level is inconsistent; for example, see two highlighted rows here. I suggest removing bolding and adjusting header level to be smaller.
![image](https://github.com/user-attachments/assets/2a3cbe8b-69d3-439b-93b9-d1a197b89ba7)